lib/linalg/dgelss.f

```diff
@@ -254,11 +254,11 @@ SUBROUTINE DGELSS( M, N, NRHS, A, LDA, B, LDB, S, RCOND, RANK,
 *
 *              Compute space needed for DGEQRF
                CALL DGEQRF( M, N, A, LDA, DUM(1), DUM(1), -1, INFO )
-               LWORK_DGEQRF=DUM(1)
+               LWORK_DGEQRF = INT( DUM(1) )
```

Comment:
Are these changes necessary. Is this like an upgrade to BLAS. If so, can we add a version number to the README file?

Reply:
I cherry-picked this from a different branch. This adds two new LAPACK functions to the linalg library that are used by the code here. The LAPACK version update was part of the change set. Including the entire change set avoids a merge conflict.

Reply:
> can we add a note to the README file with a BLAS version number?

Done in 9b7b45bdea6b4ab2359c72d94cc63fe79fb6cb52